releaser/README.md

```diff
@@ -0,0 +1,151 @@
+# Go Module Releaser
```

Comment:
Happened to find this PR - seems like an important tool for this repo, but I think the name needs to be reconsidered. It's too similar to a very popular tool

https://goreleaser.com/intro/

This is especially important if you happen to be thinking of a blog post about a Go Releaser :P

Focusing on the multi-module aspect seems best, like just `go-multimod`, (a multi version of go mod), `monorepo-tagger`, `pangaea` (or any similar abstract term referring to a combination of distinct parts), etc

For inspiration, a very similar tool picked the name `gump`, a version bumper for go

https://github.com/takashabe/gump

Reply:
That's a great point, thanks! I think a simple name like Go MultiMod is a good starting point since the meaning is pretty clear, and it's easy to spell. I will update the naming of the tool today.

Reply:
Updated to multimod! Thanks @anuraaga for the suggestion 👍